electrum/wizard.py

```diff
@@ -298,7 +298,12 @@ def needs_derivation_path(self, wizard_data: dict) -> bool:
 
     def wants_ext(self, wizard_data: dict) -> bool:
         wdata = self.current_cosigner(wizard_data)
-        return 'seed_variant' in wdata and wdata['seed_extend']
+        if 'seed_variant' in wdata and wdata['seed_extend']:
+            if wdata['seed_variant'] != 'electrum':
+                return True
+            return mnemonic.seed_type(wdata['seed']) != '2fa'  # old 2FA disallows passphrase
```

Comment:
Note that `old` type electrum seeds cannot have a passphrase either. Looks like this is another bug on master: the wizard allows setting one, but then it is eventually ignored in keystore.py. I'll add a check and raise in exception in keystore.py.